documentation/development-guide.asciidoc

```diff
@@ -15,12 +15,33 @@ This repository has the following structure:
       +-- capacity-tests - A set of capacity tests for calibrating memory settings
 ```
 
+The `templates` directory contains a template for the Service Catalog. The easiest way to install it is to use `make install-templates-in-openshift-namespace`.
+
+The `modules` directory contains modules used in `image.yaml`. See http://concreate.readthedocs.io/en/develop/[Concreate] documentation for more details.
+
+The `functional-tests` directory contains http://arquillian.org/arquillian-cube/[Arquillian Cube tests]. The tests assume there is a running OpenShift cluster and `oc` or `kubectl` is already attached to it.
+
+== Building the image
 The `image.yaml` uses a Python framework called http://concreate.readthedocs.io/en/develop/[Concreate] to build an image. The easiest way to start the build is to invoke `concreate image.yaml target` and invoke a standard container build from the `target` directory.
 
-The `Makefile` contains lots of small, useful scripts.
+Before using concreate, you should create a `.concreate` file in your home directory with the following content:
+```
+[common]
+ssl_verify = False
```

Comment:
Just thinking - shouldn't those changes go to https://docs.google.com/document/u/1/d/1gSGmytIW3u4IR1tXGmEekEa1AzhKfNvAPOAP0Fh32kQ/edit?usp=sharing ?